### ISSUE TITLE
adding devextreme to angular 6 + cli 6

### ISSUE THREAD

**Issue by SanderElias** [DevExpress/devextreme-angular#775]
devexteme version:  "^17.2.7",
    devextreme-angular version: "^17.2.7",
(tried with 18 too)

- Specify the type of the issue (check one with "x"):
[x] bug Report 
[ ] feature request 
[ ] support inquiry 

If I follow the instructions and add the css files to the `angular.json`, this is what happens on build:
![image](https://user-images.githubusercontent.com/1249083/39625258-f220896a-4f9c-11e8-9f62-cdf4fe921480.png)

steps to reproduce:
```
ng new demo
cd demo
yarn add  devextreme devextreme-angular
// open editor and add css files to `angular.json` 
ng build
```
My Angular version:
```
Angular CLI: 6.0.0
Node: 8.11.1
OS: win32 x64
Angular: 6.0.0
... animations, cli, common, compiler, compiler-cli, core, forms
... http, language-service, platform-browser
... platform-browser-dynamic, router

Package                           Version
-----------------------------------------------------------
@angular-devkit/architect         0.6.0
@angular-devkit/build-angular     0.6.0
@angular-devkit/build-optimizer   0.6.0
@angular-devkit/core              0.6.0
@angular-devkit/schematics        0.6.0
@ngtools/webpack                  6.0.0
@schematics/angular               0.6.0
@schematics/update                0.6.0
rxjs                              6.1.0
typescript                        2.7.2
webpack                           4.6.0
```

How can I prevent this error?

**Comment by GoshaFighten** [comment]
The root Angular CLI folder is now your project folder. So, you don't need to use '../' any more. We updated the [Using the DevExtreme Angular Components with Angular CLI](https://github.com/DevExpress/devextreme-angular/blob/master/docs/using-angular-cli.md) accordingly.

**Comment by SanderElias** [comment]
@GoshaFighten Thanks, that got it fixed, but it then directly landed me on: #776
Perhaps it's worth to mention that in the docs too? for the time it's still needed?
